DataMigration/src/test/java/org/sagebionetoworks/dian/datamigration/SynapseTests.java

```diff
@@ -0,0 +1,307 @@
+
+        String testSessionsChecksumActual = MD5ChecksumHelper.getMD5Checksum(testSessionsZipFile);
+        // Expected was calculated by downloading the zip separately in a browser and getting the checksum
+        assertEquals("cda0a862c9e90dacbe0ba33c4caecaaf", testSessionsChecksumActual);
+    }
+
+    @Test
+    public void testDownload_SessionSchedule_Zip() throws SynapseException, IOException {
+        EntityChildrenRequest request = new EntityChildrenRequest();
+        request.setParentId(project.getId());
+        request.setIncludeTypes(Lists.newArrayList(EntityType.folder));
+        EntityChildrenResponse response = synapse.getEntityChildren(request);
+
+        // NavigateFolders
+        EntityHeader header = response.getPage().get(0);
+        EntityChildrenRequest subFolderRequest = new EntityChildrenRequest();
+        subFolderRequest.setParentId(header.getId());
+        subFolderRequest.setIncludeTypes(Lists.newArrayList(EntityType.folder));
+        EntityChildrenResponse subFolderResponse = synapse.getEntityChildren(subFolderRequest);
+        EntityHeader headerTestSession = subFolderResponse.getPage().get(1);
+
+        // Download ZIPs
+        EntityChildrenRequest sessionScheduleZipRequest = new EntityChildrenRequest();
+        sessionScheduleZipRequest.setParentId(headerTestSession.getId());
+        sessionScheduleZipRequest.setIncludeTypes(Lists.newArrayList(EntityType.file));
+
+        EntityChildrenResponse sessionScheduleZipResponse = synapse.getEntityChildren(sessionScheduleZipRequest);
+        assertNotNull(sessionScheduleZipResponse);
+        assertNotNull(sessionScheduleZipResponse.getPage());
+        assertEquals(1, sessionScheduleZipResponse.getPage().size());
+
+        String zipId = sessionScheduleZipResponse.getPage().get(0).getId();
+        String zipFilename = sessionScheduleZipResponse.getPage().get(0).getName();
+
+        FileEntity fileEntity = synapse.getEntity(zipId, FileEntity.class);
+        String fileHandleId = fileEntity.getDataFileHandleId();
+        FileHandleAssociation fileHandleAssociation = new FileHandleAssociation();
+        fileHandleAssociation.setFileHandleId(fileHandleId);
+        fileHandleAssociation.setAssociateObjectId(zipId);
+        fileHandleAssociation.setAssociateObjectType(FileHandleAssociateType.FileEntity);
+
+        File testSessionsZipFile = File.createTempFile("TestSessions.zip", null);
+        testSessionsZipFile.deleteOnExit();
+        synapse.downloadFile(fileHandleAssociation, testSessionsZipFile);
+
+        String testSessionsChecksumActual = MD5ChecksumHelper.getMD5Checksum(testSessionsZipFile);
+        // Expected was calculated by downloading the zip separately in a browser and getting the checksum
+        assertEquals("c9a015e430f822d175a729b6c5733184", testSessionsChecksumActual);
+    }
+
+    @Test
+    public void testDownload_WakeSleep_Zip() throws SynapseException, IOException {
```

Comment:
Not clear what testDownload_WakeSleep_Zip is testing. There's no reference to the logic in SynapseUtil and this code need not test that the Synapse client works.